BadHabbit/BadHabbit/SceneDelegate.swift

```diff
@@ -22,10 +21,20 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         }
         
         let window = UIWindow(windowScene: windowScene)
-        
+
         self.appAssembly = AppAssembly(application: .shared, window: window)
-        window.rootViewController = appAssembly.storiesAssembly.signUpScreen.assembleStory()
-        
+
+        let navigationController = UINavigationController(rootViewController: appAssembly.storiesAssembly.signUpScreen.assembleStory())
+
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithTransparentBackground()
+        appearance.shadowColor = UIColor(red: 0.235, green: 0.235, blue: 0.263, alpha: 0.35)
```

Comment:
add UIColor to the Assets folder and use the Assets.Colors enum instead

Reply:
resolved